### ISSUE TITLE
Many interface controls in Stoat desktop app for Windows are unlabelled or missing state information

### ISSUE THREAD

**Issue by dev-shark-a11y** [stoatchat/for-desktop#200]
### Many interface controls in Stoat desktop app for Windows are unlabelled or missing state information

Impact: Blocker
WCAG 2.1 Criteria: 4.1.2 - Name, Role, Value (Level A) - [https://www.w3.org/WAI/WCAG21/Understanding/name-role-value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)

### What currently happens:
When navigating with a screen reader through the Stoat desktop app, the following interface controls are either announced as link, image or button (in some case announced as an image then a link) and not what they their role is in the application. Additionally, controls that toggle do not include state information. Users of assistive technology will not be able to access/know the controls for multiple areas of the interface.

- Minimize Window
- Restore Window
- Close Window
- Home
- User
- Server_Name_Here (any server you add to desktop)
- Find new servers to join
- Settings
- Server Settings
- Add item (plus sign in chat box)
- Gif button
- Emoji button
- Send button

### What should happen:
When navigating with a screen reader through the Stoat desktop app, the interface controls as announced with their correct name and state information.

### Machine Details:

- Windows 11 Pro - Version 25H2 (OS Build 26200.7840)
- Stoat for Desktop 1.3.0 - Version 0.3.0 [38.1.2, 22.198.0, 140.0.7339.133]
- Windows Narrator
- NVDA - Version 2025.3.2
- Voice Accessnv

### Step-by-Step Instructions:
NOTE - make sure you have created a new server in the Stoat app

1. Launch the NVDA screen reader
2. Open Stoat
3. Press Tab to enter Stoat
4. Press Up arrow three times and screen reader announces link (this should be labelled as Minimize button and coded as a button).
5. Press Down arrow and screen reader announces link (this should be labelled as Maximize and coded as a button).
6. Press Down arrow and screen reader announces link (this should be labelled as Close window and coded as a button).
7. Press Down arrow and screen reader announces "link you have 0 pending friend requests" (this should be labelled as "Home you have 0 pending friend requests" and coded as a button).
8. Press Down arrow and screen reader announces "link graphic Clickable unlabelled graphic" (this should be named User_Profile_Name menu or equivalent)
9. Press Down arrow and screen reader announces "list of one item clickable same page link 01vivaix..." (this should be labelled the name of the server you created and coded as a button).
10. Press Enter to activate the server.
11. Press Down arrow and screen reader announces "create or join a server".
12. Press Down arrow and screen reader announces "link Discovery" (this should be coded as a button).
13. Press Down arrow and screen reader announces "link graphic graphic" (this is a repeat of the previous which is Discovery).
14. Press Down arrow and screen reader announces "Settings" (unable to press Enter or Spacebar to activate it).
15. Press Down arrow and screen reader announces "graphic clickable" (this is a duplicate for settings, can be activated, but is unlabelled).
16. Press Down arrow and screen reader announces the name of the server.
17. Press Down arrow and screen reader announces "button" (this should be labelled Server settings or equivalent).
18. Press Down arrow and screen reader announces "List unavailable" (this is the area where you have created any channels for the server.
19. Press Down arrow and screen reader announces "toggle main side bar" (there is no state for this toggle, so you do not know if it is expanded or collapsed).
20. Press Down arrow until screen announces "button" (this should be labelled as Add item or equivalent).
21. Press Down arrow and screen reader announces "message general edit has autocomplete multiline".
22. Press Enter to enter forms mode and type this is a test.
23. Press Escape to exit forms mode.
24. Press Down arrow and screen reader announces "button" (this should be labelled GIF or Add GIF)
25. Press Down arrow and screen reader announces "button" (this should be labelled Emoji and Add EMoji)
26. Press Down arrow and screen reader announces "button" (this should be labelled Send)

**Comment by amycatgirl** [comment]
The desktop app uses electron to render stoat.chat/app under the hood, so I think this issue should be reopened in [stoatchat/for-web](https://github.com/stoatchat/for-web) instead, since none of the issues described above seem to be issues with the desktop app itself.

**Comment by dev-shark-a11y** [comment]
@amycatgirl - What about the "links" for minimize, maximize and close window in the upper right corner of the desktop app window. Those do not have labels. The rest of the documented areas are shared between the web app and desktop app.

**Comment by amycatgirl** [comment]
Also part of the web app, however those only get shown when the web app detects it is running inside the desktop application: https://github.com/stoatchat/for-web/blob/c6bce055d71c88f8c44eeaed89f00b579c741d23/packages/client/components/app/interface/desktop/Titlebar.tsx#L20
